File: packages/boozsoft-auth/src/main/java/org/boozsoft/repo/stock/StockCcodeRepository.java
package org.boozsoft.repo.stock;

import org.boozsoft.domain.entity.stock.StockCcode;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import reactor.core.publisher.Flux;

public interface StockCcodeRepository extends ReactiveCrudRepository<StockCcode, String> {

    Flux<StockCcode> findAllByOrderById();

}

File: packages/boozsoft-platform-zhongzhang/src/main/kotlin/org/boozsoft/repo/stock/StockCcodeRepository.kt
package org.boozsoft.repo.stock

import org.boozsoft.domain.entity.stock.StockCcode
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import reactor.core.publisher.Flux

interface StockCcodeRepository : ReactiveCrudRepository<StockCcode, String> {
    fun findAllByOrderById(): Flux<StockCcode>
}